conf/messages

```diff
@@ -118,6 +118,7 @@ error.required.parentsOccupation = What was your parent(s) or guardian(s) work s
 error.required.employee = Were your parent(s) or guardian(s) employed or self-employed?
 error.required.organizationSize = What was the size of the organisation?
 error.required.supervise = Were your parent(s) or guardian(s) supervising employees?
+error.required.acceptance = There is a problem on this page. Confirm you understand that the information you give here won''t affect your application.
```

Comment:
For sure this a mistake in the story description and we shouldn't have error messages like that. It doesn't fit to the other messages.